src/main/java/org/broadinstitute/hellbender/utils/test/SamAssertionUtils.java

```diff
@@ -97,6 +98,7 @@ public static void assertSamsEqualLenient(final File actualSam, final File expec
      * @param reference is allowed to be null
      */
     public static void assertSamValid(final File sam, final ValidationStringency validationStringency, final File reference) throws IOException {
+        assertValidateContentsIfCRAM(sam, reference);
```

Comment:
Name `assertValidateContentsIfCRAM()` is a bit awkward and confusing -- how about `assertCRAMContentsIfCRAM()`